### PR TITLE
Production 2024-03-19_01

### PR DESCRIPTION
**Pull request by pavelbrm**
- #2395
- #2426
- #2427

### CODE REVIEW

**Review by porteron — DISMISSED** [review]
Code looks straightforward and the tests have coverage 👍

File: services/skus/service_nonint_test.go [hunk 1]
```diff
@@ -837,6 +837,40 @@ func TestCreateOrderWithReceipt(t *testing.T) {
 				},
 			},
 		},
+
+		{
+			name: "successful_case_android_vpn_monnthly",
```
Comment (DJAndries) [inline services/skus/service_nonint_test.go:842]: looks like you have a typo here

File: services/skus/service_nonint_test.go [hunk 2]
```diff
@@ -805,7 +805,7 @@ func TestCreateOrderWithReceipt(t *testing.T) {
 		},
 
 		{
-			name: "successful_case",
+			name: "successful_case_android_leo_monnthly",
```
Comment (DJAndries) [inline services/skus/service_nonint_test.go:808]: typo?

**Review by DJAndries — APPROVED** [review]

File: services/skus/service_nonint_test.go (hunk 1 above)
Comment (pavelbrm) [reply services/skus/service_nonint_test.go:842]: Good spotting, thanks, fixed in https://github.com/brave-intl/bat-go/pull/2426.

File: services/skus/service_nonint_test.go (hunk 2 above)
Comment (pavelbrm) [reply services/skus/service_nonint_test.go:808]: Thanks, fixed in https://github.com/brave-intl/bat-go/pull/2426.

**Review by DJAndries — APPROVED** [review]

**Pull request merged**